src/global-firestore-data/onboarding-data.ts

```diff
@@ -0,0 +1,30 @@
+import { onboardingDataCollection } from '@/firebase-frontend-config';
+import store from '@/store';
+import setUsernameData from './username-data';
+import { clearOverridenRequirementsAPIB } from './overridden-fulfillment-choices';
+
+const setOnboardingData = (name: FirestoreUserName, onboarding: AppOnboardingData): void => {
+  setUsernameData(name);
```

Comment:
I split up `setUsernameData` and put it in `./username-data`.

Reply:
Do we think we'll have more logic for `username-data`? It kinda feels like a file for it is excessive.

Reply:
Idk tbh. If not, maybe a separate collection for it is also excessive?

Maybe we can remove it in the future if we choose to remove the username collection and merge it into the onboarding collection.

Reply:
ahhh gotcha!